LinkerScript.sublime-syntax

```diff
@@ -72,7 +76,7 @@ contexts:
               |
                 \B\.[A-Za-z_\.0-9\-]+
               )'
-      scope: variable.other.linker
+      scope: entity.name
```

Comment:
According to Sublime Text's [scope naming documentation](https://www.sublimetext.com/docs/3/scope_naming.html), the "entity." scope is more for types, structs, etc., so I think the "variable." scope is better here.